cypher/cypher-docs/src/docs/dev/ql/administration/security/index.asciidoc

```diff
@@ -46,6 +46,7 @@ This section explains how to use Cypher to manage Neo4j role-based access contro
 ** <<administration-security-limitations-labels, Security and labels>>
 ** <<administration-security-limitations-db-operations, Security and count store operations>>
 
+
```

Comment:
This is very inconsistent in all our administration files, some places have 2 and some have 1 blank lines, would probably benefit from going an overall sweep and make it consistent

Reply:
Yep, we actually have review + restructure this section as an action on our wall - we are just looking for an opportune moment to do it, as it would likely take us around 2-3 weeks and would need to freeze any submissions while we do it! Maybe in scope for the last few weeks of the year though